plugins/modules/ise_radius_integration_workflow_manager.py

```diff
@@ -1368,6 +1370,49 @@ def check_auth_server_response_status(self, response, validation_string_set, api
 
         return self
 
+    def check_ise_server_updation_status(self, have_auth_details, want_auth_details):
+        """
+        Check if the Cisco ISE server requires an update or not by checking the user name,
+        fqdn value and the state of the Cisco ISE server.
+
+        Parameters:
+            have_auth_details (dict) - Authentication and policy server information from the Cisco Catalyst Center.
+            want_auth_details (dict) - Authentication and policy server information from the Playbook.
+        Returns:
+            True or False (bool): Return True, if the Cisco ISE server requires an update. Else, False.
+        Description:
+            Check the user name and the fully qualified domain name between the config provided by the
```

Comment:
Compares the user name and FQDN between the existing configuration and the new configuration.
        If there is a discrepancy, or if the server state is not 'ACTIVE', an update is required.